src/extlibs/hrvo/BUILD

```diff
@@ -0,0 +1,31 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
please add a readme in this folder pointing to the hrvo repo

Reply:
README added